src/me/mrCookieSlime/Slimefun/Android/ProgrammableAndroid.java

```diff
@@ -700,36 +706,72 @@ private void move(Block b, BlockFace face, Block block) throws Exception {
 		}
 	}
 
-	private void mine(Block b, Block block) {
+	private void mine(Block b, Block block, SlimefunItem item) {
```

Comment:
You could also just declare

SlimefunItem item = BlockStorage.check(block);

inside both methods.